applications/launchpad/docker_rig/tarilabs.Dockerfile

```diff
@@ -0,0 +1,194 @@
+# syntax = docker/dockerfile:1.3
+#FROM quay.io/tarilabs/rust_tari-build-with-deps:nightly-2021-11-01 as builder
+
+# Declare TARGETARCH to make it available
+#ARG TARGETARCH
+
+FROM rust:1.60-bullseye as builder
+
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGETVARIANT
+
+#RUN apt update && apt-get --no-install-recommends install -y \
+
+# Disable anti-cache
+RUN rm -f /etc/apt/apt.conf.d/docker-clean; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
+# https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md#run---mounttypecache
+RUN --mount=type=cache,id=build-apt-cache-${TARGETOS}-${TARGETARCH}${TARGETVARIANT},sharing=locked,target=/var/cache/apt \
+    --mount=type=cache,id=build-apt-lib-${TARGETOS}-${TARGETARCH}${TARGETVARIANT},sharing=locked,target=/var/lib/apt \
+  apt update && apt-get install -y \
+  apt-transport-https \
+  bash \
+  ca-certificates \
+  curl \
+  gpg \
+  iputils-ping \
+  less \
+  libreadline-dev \
+  libsqlite3-0 \
+  openssl \
+  telnet \
+  cargo \
+  clang \
+  cmake
+
+WORKDIR /tari
+
+# Adding only necessary things up front and copying the entrypoint script last
+# to take advantage of layer caching in docker
+ADD Cargo.lock applications/deps_only/Cargo.lock
+ADD rust-toolchain.toml .
+
+#RUN pwd
+#RUN ls
+#RUN ls /
+#RUN ls /tari
```

Comment:
remove.

Reply:
Yip, we can remove. This PR should have been marked a WIP, as I was looking for feedback on bits. Can remove testing comments.